src/test/functional/tests/loginPage.spec.ts

```diff
@@ -16,21 +14,17 @@ test.describe("Login Page", () => {
     await loginPage.close();
   });
 
-  test("Test Case 1: Login User", async ({ page }) => {
+  test("Test Case 1: Login As An Admin", async ({ page }) => {
     const loginPage = new LoginPage(page);
     await loginPage.navigateToMainPage()
-    await loginPage.loginUser(userTestData.name, userTestData.password)
-    await expect(page.locator(loginPageLocator.userNameAfterLogin)).toHaveText(loginPageAssertion.userName)
-    // await page.waitForTimeout(5000); dodane na ta chwile w celach debugowych
+    await loginPage.loginUser(userTestData.adminName, userTestData.password)
+    await expect(loginPage.userNameAfterLogin).toHaveText(loginPageAssertion.adminName);
   });
 
-
-
-
-
-
-
-
-
-
+  test("Test Case 2: Login As A User", async ({ page }) => {
+    const loginPage = new LoginPage(page);
```

Comment:
You create login page in the beforeEach function so I guess we can avoid creating it again in each test?

Reply:
yes i know but maybe in future we will skip logging in beforeEach?